src/Scene.cpp

```diff
@@ -174,5 +174,9 @@ namespace RamsesPython
 
         return "";
     }
-}
 
+    ramses::RamsesObject* Scene::findObjectByName(const char* name)
```

Comment:
You have to wrap the pointer in a wrapper object - same as all other wrapper objects here. Why? Because otherwise Python might decide to just delete [] the RamsesObject* - which will cause a crash in ramses, because ramses objects are destroyed by calling Scene::destroy(RamsesObject* object), not by directly deleting the memory.